src/test/unitTests/authentication.test.js

```diff
@@ -0,0 +1,56 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+// eslint-disable-next-line import/no-unresolved
+import { shuffleToken, generateToken, verifyToken } from '../../utils/authentication';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const payload = {
+  email: 'johndoe23@gmail.com',
+  password: '1234567'
+};
+
+describe('Tests for the authentication helper methods', () => {
+  describe('should generate a token', () => {
+    it('should return a shuffled token', () => {
+      const token = generateToken(payload);
+      expect(token).to.be.a('string');
+    });
+  });
+
+  describe('should shuffle token', () => {
+    it('should shuffle the generated token and return it to it\'s original state', () => {
+      const token = generateToken(payload);
+      const initialToken = shuffleToken(token);
+      expect(initialToken).to.be.a('string');
+    });
```

Comment:
assert for the `initialToken` to resemble `payload`. i.e both properties of `payload` and `initialToken` should be identical